src/fluree/db/query/fql/parse.cljc

```diff
@@ -366,14 +368,37 @@
     (parse-variable id)
     (parse-subject-iri id context)))
 
+(declare parse-predicate)
+(defn parse-property-path
```

Comment:
There currently isn't a clojure friendly property path syntax. What's wrong with something like `[:ex/foo :+]`?

Reply:
I'm not sure that there's anything wrong with that, I was just trying to keep parsing as simple as I could - once you start composing path operators it can get pretty gnarly and I wanted to avoid introducing divergent behavior at this stage.